src/System.Windows.Forms.Primitives/src/Interop/WinFormsComWrappers.cs

```diff
@@ -114,6 +115,26 @@ internal unsafe partial class WinFormsComWrappers : ComWrappers
             return wrapperEntry;
         }
 
+        private static ComInterfaceEntry* InitializeAccessibleObjectEntry()
+        {
+            GetIUnknownImpl(out IntPtr fpQueryInterface, out IntPtr fpAddRef, out IntPtr fpRelease);
+
+            IntPtr iRawElementProviderSimpleVtbl = IRawElementProviderSimpleVtbl.Create(fpQueryInterface, fpAddRef, fpRelease);
+            IntPtr iRawElementProviderFragmentVtbl = IRawElementProviderFragmentVtbl.Create(fpQueryInterface, fpAddRef, fpRelease);
+            IntPtr iRawElementProviderFragmentRootVtbl = IRawElementProviderFragmentRootVtbl.Create(fpQueryInterface, fpAddRef, fpRelease);
+
+            ComInterfaceEntry* wrapperEntry = (ComInterfaceEntry*)RuntimeHelpers.AllocateTypeAssociatedMemory(
+                typeof(WinFormsComWrappers),
+                sizeof(ComInterfaceEntry) * 3);
+            wrapperEntry[0].IID = IID.IRawElementProviderSimple;
+            wrapperEntry[0].Vtable = iRawElementProviderSimpleVtbl;
+            wrapperEntry[1].IID = IID.IRawElementProviderFragment;
+            wrapperEntry[1].Vtable = iRawElementProviderFragmentVtbl;
+            wrapperEntry[2].IID = IID.IRawElementProviderFragmentRoot;
+            wrapperEntry[2].Vtable = iRawElementProviderFragmentRootVtbl;
+            return wrapperEntry;
```

Comment:
To increase readability
```suggestion
            ComInterfaceEntry* wrapperEntry = (ComInterfaceEntry*)RuntimeHelpers.AllocateTypeAssociatedMemory(
                typeof(WinFormsComWrappers),
                sizeof(ComInterfaceEntry) * 3);

            wrapperEntry[0].IID = IID.IRawElementProviderSimple;
            wrapperEntry[0].Vtable = iRawElementProviderSimpleVtbl;

            wrapperEntry[1].IID = IID.IRawElementProviderFragment;
            wrapperEntry[1].Vtable = iRawElementProviderFragmentVtbl;

            wrapperEntry[2].IID = IID.IRawElementProviderFragmentRoot;
            wrapperEntry[2].Vtable = iRawElementProviderFragmentRootVtbl;

            return wrapperEntry;
```